docs/sphinx/user_guide/build.rst

```diff
@@ -210,3 +210,44 @@ sizes. The CMake option for this is
 
 will build versions of GPU kernels that use 64, 128, 256, 512, and 1024 threads
 per GPU thread-block.
+
+Building with Caliper
+---------------------
+
+RAJAPerf Suite may also use Caliper instrumentation, with per variant output into 
+Spot/Hatchet .cali files. Original timing is nested within Caliper annotations and 
+so is not impacted when Caliper support is turned on. While Caliper is low-overhead
+it is not zero, so it will add a small amount of timing skew in its data as 
+compared to the original. For much more on Caliper, read it's documentation here:
+[Caliper] (http://software.llnl.gov/Caliper/)
+
+Caliper *annotation* is in the following tree structure
+
+Variant
+   Group
+     Kernel
+       Kernel.Tuning
+
+| Build against these Caliper versions
+|
+|   **caliper@2.9.0** (preferred target)
```

Comment:
Can we add checks for this in the top-level CMakeLists.txt file? These constraints should also be contained in the RAJAPerf Spack package.